job_bundles/tile_render_maya_ffmpeg/template.yaml

```diff
@@ -0,0 +1,331 @@
+specificationVersion: jobtemplate-2023-09
+name: tile render maya ffmpeg
```

Comment:
This is a great example, thanks for putting it together!

The PR summary covers a bit about what the job is for, but it would be great to have that right here in the template. A `description: ...` would be awesome to have here, that describes what the tiled job does, and why you may want to run a job as tiled.

Can you also adjust the name to match the pattern, like a document title, that the other examples follow? Something similar to "Maya Tiled Render"